handsontable/src/3rdparty/walkontable/css/walkontable.scss

```
@@ -90,18 +90,22 @@
 
 .handsontable th:last-child {
   /*Foundation framework fix*/
-  border-right: 1px solid #ccc;
+  border-inline-end: 1px solid #ccc;
   border-bottom: 1px solid #ccc;
 }
 
 .handsontable th:first-child,
 .handsontable th:nth-child(2),
 .handsontable td:first-of-type {
-  border-left: 1px solid #ccc;
+  border-inline-start: 1px solid #ccc;
 }
 
+.handsontable .ht_clone_top th:nth-child(2) {
```

Nice job 👏  

Old code from <=2015 year has been removed. WDYT about an extra comment? For me, iit may be unclear what the code does.

```suggestion

/* It removes double right border from first column header when row headers are disabled */
.handsontable .ht_clone_top th:nth-child(2) {
```